src/js/components/FormPasswordInput/FormPasswordInput.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
 import { FormInput, Button } from '../..';
+import translation from '../../locale/i18n';
```

Comment:
It might be more suitable using hooks here, if it works:
`import { useTranslation } from 'react-i18next';`
`const { t } = useTranslation();`